### DISCUSSION TITLE
Issue #309 needs to be reopened, it is still a problem

### DISCUSSION THREAD

**Discussion by DavidRBailey** [hassio-addons/addon-zwavejs2mqtt#399] [Q&A] [answered]
Issue #309 was closed by the github-actions bot, but it needs to be reopened, it is still a problem.

How can this be accomplished?

**Comment by frenck** [comment, marked as answer]
No. The bot asked to comment if the issue was still relevant. Nobody did. Bot closed it.

**Comment by DavidRBailey** [comment]
So regardless of that, it is still an issue and needs to be reopened. Is a new ticket required?
